packages/mdl-fab/index.scss

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-fab {
+  box-sizing: border-box;
+  display: inline-block;
+  width: 56px;
+  height: 56px;
+  padding: 0;
+  border: none;
+  border-radius: 50%;
+  background-color: var(--mdl-accent-color);
+  color: white;
+  text-align: center;
+  box-shadow: 0 1px 1.5px 0 rgba(0, 0, 0, .12), 0 1px 1px 0 rgba(0, 0, 0, .24);
+  cursor: pointer;
+  overflow: hidden;
+  -moz-user-select: none;
```

Comment:
This prevents copying/selection of the FAB. Users shouldn't need to copy it ever, so this helps prevent a double-click causing a selection of text for instance if they get it right on the icon. Even though the icon is `pointer-events: none;` now, this is still useful as a safety-net.

Can't not use prefixes for this one, as `user-select` isn't a thing yet for anyone and last I tried autoprefixer does't handle it.